vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util_unittest.cc

```diff
@@ -36,6 +38,16 @@ TEST(PaymentUtilTest, GetServerUrlProduction) {
   ASSERT_EQ(url, "https://payment.rewards.brave.com/test");
 }
 
+TEST(PaymentUtilTest, GetServerUrlWithPaymentServiceURLSwitch) {
```

Comment:
only one unit test added? There's no way that is sufficient for a PR of this size, particularly one that involves payments. The tests (both unit and browser) need to be beefed up substantially

Reply:
resolved, browsertests added: https://github.com/brave/brave-core/pull/7557/files#diff-3bf39fcf0424a5290d52c433dd5841175d441396839895f8af35cd8426a1b3ff